### PR TITLE
adding 1.3.0 release notes to changelog

### PR DESCRIPTION
**Pull request by wallentx**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,78 +6,89 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project does not yet adhere to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 for setuptools_scm/PEP 440 reasons.
 
-## 1.3.0 Beta Chia blockchain 2022-2-10
+## 1.3.0 Chia blockchain 2022-3-07
 
-We at Chia have been working hard to bring all of our new features together into one easy-to-use release. Today, we’re proud to announce the beta release of our 1.3 client. Because this release is still in beta, we recommend that you only install it on non mission-critical systems. If you are running a large farm, you should wait for the full 1.3 release before upgrading. When will the full version of 1.3 be released? Soon.
+We at Chia have been working hard to bring all of our new features together into one easy-to-use release. Today, we’re proud to announce the release of our 1.3.0 client. There is a lenghty list of added features and changes, as well as a few noteworthy items to be aware of when upgrading. We'd also like to give a very special thanks to our beta testers! Thanks to their efforts, we've found, and remediated may bugs during the beta phases of this release.
```
Comment (paninaro) [inline CHANGELOG.md:11]: typo: `lenghty`

Comment (paninaro) [reply CHANGELOG.md:11]: typo: `may` instead of `many`

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,78 +6,89 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
 ### Changed:
 
-- Light wallet client sync updated to only require 3 peers instead of 5
-- Only CATs from the default CAT list will be automatically added, all other unknown CATs will need to be manually added
-- New sorting pattern for offer history - Open/pending offers sorted on top ordered by creation date > confirmation block height > trade id, and then Confirmed and Cancelled offers sorted by the same order
-- When plotting multiple plots with the GUI, new items are taken from the top of the list instead of the bottom
-- CA certificate store update
-- VDF, chiapos, and blspy workflows updated to support python 3.10 wheels
+- Light wallet client sync updated to only require 3 peers instead of 5.
+- Only CATs from the default CAT list will be automatically added, all other unknown CATs will need to be manually added (this behavior can be toggled in config.yaml).
```
Comment (paninaro) [inline CHANGELOG.md:36]: Might be worth crediting ojura for part of this (specifically the config.yaml toggle)
https://github.com/Chia-Network/chia-blockchain/commit/05f96670181654415b1ad219d38463228f30b0ce

File: CHANGELOG.md (hunk 1 above)
Comment (wallentx) [reply CHANGELOG.md]: > typo: `lenghty`

fixed

> typo: `may` instead of `many`

fixed

File: CHANGELOG.md (hunk 2 above)
Comment (wallentx) [reply CHANGELOG.md]: Added

**Review by justinengland — APPROVED** [review]

**Pull request merged**